### DISCUSSION TITLE
Quick way to wrap element with container

### DISCUSSION THREAD

**Discussion by szymonmil** [elementor/elementor#29590] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Often i must wrap some element with container (common use case from writing in native HTML). For example I have title which i want to wrap with container. Now there is no quick way to do that - i must place container next to element and move that element into container. It's not always that easy (for example with complex structure)

### Describe the solution you'd like.

It will be very helpful where there will be some way to do it quickly.
For example in Navigator menu, when i select some element, there should be option "Wrap with container" or something like that. That also could be a shortcut for this.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by ryanmish** [comment]
I came here to make this exact request, however instead of Navigator menu, I think it should be in the right click menu. This is something common in other builders like FlutterFlow.

**Comment by zuzanu** [comment]
There has already been a request like this for over a year: https://github.com/orgs/elementor/discussions/24311

**Comment by vovkasolovev** [comment]
I agree. Very often I want to press something common like _Ctrl+G_ to **wrap the selected widget in a container**. Since Elementor now allows you to select multiple widgets, it would be great if they all went into the container if the widgets are siblings.

And of course, we need a reverse shortcut: to remove from the container if you select the container. It is also often desirable to use an action familiar from other design applications, such as _Ctrl+Shift+G_.
